### ISSUE TITLE
Some links are not recognized

### ISSUE THREAD

**Issue by bartaz** [alex-page/get-site-urls#14]
Hi,

I noticed that some links on our website were not recognized and crawled by `get-site-urls`.
In our case it was the links that had new lines in the href, like so:

```html
<a href="
   /example
">Example</a>
```

These were generated by static site generator.
Trimming the URLs fixed the issue, but having new lines inside href is completely valid HTML, so it shouldn't prevent `get-site-urls` from recognizing the link.

While debugging this I also found that links that use single quotes are not recognized:

```html
<a href='/example'>Example</a>
```

Possibly there are other (HTML valid) cases that are not working in `get-site-urls`.

**Comment by alex-page** [comment]
Hey @bartaz, I have a test for URL's like this. If you can figure out what is different I would love to know!

https://github.com/alex-page/get-site-urls/blob/e5c7f846fc7e968059eaf971b73656cf58de0d63/test.js#L71

**Comment by bartaz** [comment]
We fixed it on our side, by getting rid of unnecessary whitespace and new lines, but still - it's valid HTML that silently fails.

I guess it would be more important to fix `<a href='single quote'>` and `<a href=no-quote>` as they definitely may be more common than new lines in links.

**Comment by alex-page** [comment]
@bartaz I am shipped v2.0.0 with a bunch of improvements. Feel free to reopen this issue if there are problems.
